instagram-challenge/app/views/users/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= @user.name %>
+
+<%= link_to 'Sign out', sessions_new_path%>
```

Comment:
why would a link to sign out go to sessions new? Surely that is for sign in? no?